generator/README.md

```diff
@@ -24,6 +24,19 @@ The generator reads in from `generator.yml` and writes to `snmp.yml`.
 
 Additional command are available for debugging, use the `help` command to see them.
 
+## Docker Users
+
+If you would like to run the generator in docker to generate your `snmp.yml` config run the following commands.
+
+```
+docker build -t snmp-generator .
+docker run -ti \
+  -v $PWD/mibs:/root/.snmp/mibs \
+  -v $PWD/generator.yml:/opt/generator.yml:ro \
+  -v $PWD/out/:/opt/build \
```

Comment:
Why put it in a subdir of the cwd rather than directly in the cwd?

Reply:
good call - making this change